vehicle/remote_cmd_converter/src/node.cpp

```diff
@@ -16,39 +16,62 @@
 
 #include "remote_cmd_converter/node.hpp"
 
-RemoteCmdConverter::RemoteCmdConverter() : nh_(""), pnh_("~")
+using std::placeholders::_1;
+
+RemoteCmdConverter::RemoteCmdConverter()
+: Node("remote_cmd_converter"),
+  ros_clock_(RCL_ROS_TIME),
```

Comment:
I think the clock can be removed if you call `this->now()` instead. It will also respect ROS time (whereas I'm not sure what the behavior of this clock is when sim time is disabled).

Reply:
Done.